packages/scripts/src/utils/contracts.ts

```diff
@@ -1,4 +1,4 @@
-const { getAbi, getAddress } = require("@uma/contracts-node");
+const { getAddress } = require("@uma/contracts-node");
```

Comment:
nit: can we use import for this and the hre line below?

Reply:
ye, that's a good point given this is ts.